manim/utils/iterables.py

```diff
@@ -192,8 +227,8 @@ def make_even(iterable_1: Iterable, iterable_2: Iterable) -> tuple[list, list]:
 
 
 def make_even_by_cycling(
-    iterable_1: Collection, iterable_2: Collection
-) -> tuple[list, list]:
+    iterable_1: Collection[T], iterable_2: Collection[U]
+) -> tuple[list[T | U], list[T | U]]:
```

Comment:
```suggestion
) -> tuple[list[T], list[U]]:
```
According to the implementation. Am I right?

Reply:
Yup, changed it.